tests/components/line-series-canvas-test.js

```diff
@@ -0,0 +1,51 @@
+import test from 'tape';
+import React from 'react';
+import {mount} from 'enzyme';
+import XYPlot from 'plot/xy-plot';
+import LineSeriesCanvas from 'plot/series/line-series-canvas';
+
+const LINE_PROPS = {
+  className: 'line-chart-example',
+  color: '#12939a',
+  data: [
+    {x: 1, y: 5, y0: 6},
+    {x: 2, y: 20, y0: 11},
+    {x: 3, y: 10, y0: 9}
+  ]
+};
+
+test('LineSeriesCanvas: should be rendered', t => {
+  const k = Math.round((Math.random() + 1) * 5);
```

Comment:
Couple of things here.
0. I super applaud writing tests for these components, I'd be dragging my feet on that for a long time
1. when writing tests it is a bad practice to use random values as this can to inconsistent test behaviour.
2. Generally the pattern that is followed in our tests is to write a basic props test (like this one) and then write tests against showcase components, that way the UI can be inspected if something is confused. Would you mind turning your second test into a showcase component (also with non random values)

Reply:
ok got it, for some reason i thought that random stuff in tests better because set of states tests can reach becomes larger, will make it 7 instead of random, what do you mean by showcase ? it like demo component ? if my boss give me some free time on weekend i will do it, i can even add some more tests to this component i remember there was some other issue, will look into this

Reply:
i just tried to create showcase but was unable to compile showcase build, i created issue for this, sorry :)